tests/integration.rs

```diff
@@ -1113,13 +1113,14 @@ mod tests {
             let mut clear = ClearDiskConfig::new();
             let guest = Guest::new(&mut clear);
 
+            let vunet_tap0 = "vunet-tap0";
+
             // Start the daemon
             let mut daemon_child = Command::new("target/release/cloud-hypervisor")
                 .args(&[
                     "--net-backend",
                     format!(
-                        "ip={},mask=255.255.255.0,sock=/tmp/vunet.sock,num_queues=4,queue_size=1024",
-                        guest.network.host_ip
+                        "tap={},ip={},mask=255.255.255.0,sock=/tmp/vunet.sock,num_queues=4,queue_size=1024", vunet_tap0, guest.network.host_ip
```

Comment:
The problem here is that you modify an existing test, which modifies the previous behavior. Are we okay about testing only when the tap interface is already created? Or should we test both use cases?

Reply:
Good catch. We need to test both. There are a lot of variables here so it is probably a good idea to factorise the test like the virtio-fs ones